imports/server/accounts.ts

```diff
@@ -44,10 +44,10 @@ const summaryFromLoginInfo = function (info: LoginInfo) {
   }
 };
 
-Accounts.onLogin((info: LoginInfo) => {
+Accounts.onLogin(async (info: LoginInfo) => {
```

Comment:
`Accounts.onLogin` doesn't explicitly handle async functions, but it also doesn't explicitly block, so this seems fine.